kratos/tests/cpp_tests/geometries/test_projection_nurbs_geometry.cpp

```diff
@@ -188,20 +188,21 @@ typedef Node<3> NodeType;
         KRATOS_CHECK_NEAR(parameter[1], 0.5, TOLERANCE);
         KRATOS_CHECK_VECTOR_NEAR(projected_point, projected_point_expected1, TOLERANCE);
 
-        // Try projection of point (0.06, -0.06, 0.06) close to the singularity
+        // Try projection of point (0.1, -0.005, 0.005) close to the singularity
         point[0] = 0.1;
         point[1] = -0.005;
         point[2] = 0.005;
 
-         // Try projection with initial guess at u = 0.5, v = 0.5
+        // Try projection with initial guess at u = 0.5, v = 0.5
         parameter[0] = 0.5;
         parameter[1] = 0.5;
 
         // Try projection
-        bool isConverged2 = ProjectionNurbsGeometryUtilities::NewtonRaphsonSurface(parameter, point, projected_point, surface);
+        bool isConverged2 = ProjectionNurbsGeometryUtilities::NewtonRaphsonSurface(parameter, point, projected_point, surface, 100, 1e-12);
+        exit(-1);
```

Comment:
I do not think this should be here :confused:

Reply:
Yes indeed! Thanks